cmd/kube-apiserver/app/server.go

```diff
@@ -213,6 +213,22 @@ func newEtcd(etcdConfigFile string, etcdServerList util.StringList, storageVersi
 	return master.NewEtcdHelper(client, storageVersion, pathPrefix)
 }
 
+// Tells the service manager like systemd about startup. For further docs:
+// http://www.freedesktop.org/software/systemd/man/sd_notify.html
+func notifyReady() {
```

Comment:
Oh, you're right. I'm the confused one. I was thinking of this: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cmd/hyperkube/kube-apiserver.go

But that's not as general as I thought it was. So nevermind.

However you should consider moving this function to pkg/util or something and calling it NotifySystemdReady or some such.